independent-projects/tools/devtools-common/src/main/java/io/quarkus/platform/catalog/processor/CatalogProcessor.java

```diff
@@ -43,11 +48,8 @@ public static CatalogProcessor of(ExtensionCatalog catalog) {
                 orderedCategories.add(new ProcessedCategory(c, extsByCategory.get(c.getId())));
             }
         }
-        JsonCategory category = new JsonCategory();
-        category.setId(UNCATEGORIZED_ID);
-        category.setName(UNCATEGORIZED_NAME);
-        category.setDescription(UNCATEGORIZED_DESCRIPTION);
-        orderedCategories.add(new ProcessedCategory(category, extsByCategory.get(UNCATEGORIZED_ID)));
+        orderedCategories
+                .add(new ProcessedCategory(UNCATEGORIZED_CATEGORY, extsByCategory.get(UNCATEGORIZED_CATEGORY.getId())));
```

Comment:
Should the `UNCATEGORIZED_CATEGORY` be added always or only if an uncategorized extension exists?

Reply:
I haven't fully checked, but is it possible that the `UNCATEGORIZED_CATEGORY` is already added to `orderedCategories`? See https://github.com/quarkusio/quarkus/pull/21522/files#diff-0e29400a4f0a08e458e374f1f9e35247a4dd3f07676050efad3e82fda8fef8d0R48